string model_name
bool visible
---
bool success